python/cuml/test/test_tfidf.py

```diff
@@ -0,0 +1,99 @@
+# Copyright (c) 2019-2020, NVIDIA CORPORATION.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import pytest
+import numpy as np
+import cupy as cp
+from cuml.feature_extraction.tfidf import TfidfTransformer
+from sklearn.feature_extraction.text import TfidfTransformer as SkTfidfTransfo
+
+
+# DATAS_IDS correspond to DATAS, order is important
+DATAS_IDS = ['base_case', 'diag', 'empty_feature', '123', 'empty_doc']
+DATAS = [
```

Comment:
```suggestion
data_ds = ['base_case', 'diag', 'empty_feature', '123', 'empty_doc']
data = [
```

All capital variable names just look very odd and against most of our codebase

Reply:
Good catch.